monai/fl/client/monai_algo.py

```diff
@@ -38,32 +39,61 @@ def convert_global_weights(global_weights, local_var_dict):
     model_keys = global_weights.keys()
     for var_name in local_var_dict:
         if var_name in model_keys:
+            weights = global_weights[var_name]
             try:
+                # reshape global weights to compute difference later on
+                weights = torch.reshape(torch.as_tensor(weights), local_var_dict[var_name].shape)
                 # update the local dict
-                local_var_dict[var_name] = torch.as_tensor(global_weights[var_name])
+                local_var_dict[var_name] = weights
             except Exception as e:
                 raise ValueError(f"Convert weight from {var_name} failed.") from e
     return local_var_dict
 
 
+def compute_weight_diff(global_weights, local_var_dict):
```

Comment:
Could you please help add the doc-string for the function and args?

Thanks.

Reply:
added